core/src/apps/cardano/sign_tx.py

```diff
@@ -630,63 +644,64 @@ async def _show_stake_pool_registration_tx(
         ctx, pool_parameters, msg.network_id, msg.protocol_magic
     )
 
-    for owner in pool_parameters.owners:
-        if owner.staking_key_path:
-            await _fail_or_warn_if_invalid_path(
-                ctx,
-                SCHEMA_STAKING,
-                owner.staking_key_path,
-                POOL_OWNER_STAKING_PATH_NAME,
-            )
-
-    await confirm_stake_pool_owners(
-        ctx, keychain, pool_parameters.owners, msg.network_id
-    )
     await confirm_stake_pool_metadata(ctx, pool_parameters.metadata)
     await confirm_transaction_network_ttl(
         ctx, msg.protocol_magic, msg.ttl, msg.validity_interval_start
     )
-    await show_auxiliary_data(
-        ctx, keychain, msg.auxiliary_data, msg.protocol_magic, msg.network_id
-    )
-    await confirm_stake_pool_registration_final(ctx)
 
 
-async def _show_outputs(
-    ctx: wire.Context, keychain: seed.Keychain, msg: CardanoSignTx
-) -> int:
-    total_amount = 0
-    for output in msg.outputs:
-        if output.address_parameters:
-            await _fail_or_warn_if_invalid_path(
-                ctx,
-                SCHEMA_ADDRESS,
-                output.address_parameters.address_n,
-                CHANGE_OUTPUT_PATH_NAME,
-            )
+async def _show_pool_owner(
+    ctx: wire.Context,
+    msg: CardanoSignTxInit,
+    keychain: seed.Keychain,
+    owner: CardanoPoolOwner,
+) -> None:
+    if owner.staking_key_path:
+        await _fail_or_warn_if_invalid_path(
+            ctx,
+            (SCHEMA_STAKING,),
+            owner.staking_key_path,
+            POOL_OWNER_STAKING_PATH_NAME,
+        )
 
-            await _show_change_output_staking_warnings(
-                ctx, keychain, output.address_parameters, output.amount
-            )
+    await confirm_stake_pool_owner(ctx, keychain, owner, msg.network_id)
 
-            if _should_hide_output(output.address_parameters.address_n, msg.inputs):
-                continue
 
-            address = derive_human_readable_address(
-                keychain, output.address_parameters, msg.protocol_magic, msg.network_id
-            )
-        else:
-            assert output.address is not None  # _validate_outputs
-            address = output.address
+def _validate_witness(
```

Comment:
I would expect this to check the derivation path (that it is 1852 etc.). If it is done elsewhere, let's mention it here in a comment / docstring.